src/app/courses/components/course-card/course-card.component.spec.ts

```diff
@@ -20,4 +31,73 @@ describe('CourseCardComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('is onCourseDelete method exists', () => {
+    const spy = spyOn(component, 'onCourseDelete');
+    component.onCourseDelete();
+    expect(spy).toHaveBeenCalled();
+  });
+
+  it('is onCourseEdit method exists', () => {
```

Comment:
In general, the name should describe what you expect from this test. As far as you want to test if the method was called it should be something like this: 'should call onCourseEdit method'.